packages/world/test/MudTest.t.sol

```diff
@@ -2,10 +2,10 @@
 pragma solidity >=0.8.0;
```

Comment:
just realized this and other tests here are still using `0.8.0`, should we update to `0.8.21`?